### PR TITLE
Clarify what "tiered" means

### PR DESCRIPTION
**Pull request by LilithHafner**
The tier that preserves the most version information is always `all`, but `tiered` doesn't always use that tier. This clarifies exactly when it doesn't.

And bold `tiered` because it is the default.

### CODE REVIEW

File: src/REPLMode/command_declarations.jl [hunk 1]
```diff
@@ -123,13 +123,14 @@ If the package is not located at the top of the git repository, a subdirectory c
...
-| `semver` | Preserve semver-compatible versions of direct dependencies                          |
-| `none`   | Do not attempt to preserve any version information                                  |
-| `tiered` | Use the tier which will preserve the most version information (this is the default) |
+| Argument     | Description                                                                        |
+|:-------------|:-----------------------------------------------------------------------------------|
+| `all`        | Preserve the state of all existing dependencies (including recursive dependencies) |
+| `direct`     | Preserve the state of all existing direct dependencies                             |
+| `semver`     | Preserve semver-compatible versions of direct dependencies                         |
+| `none`       | Do not attempt to preserve any version information                                 |
+| **`tiered`** | Use the tier which will preserve the most version information                      |
```
Comment (LilithHafner) [inline src/REPLMode/command_declarations.jl:132]:
```suggestion
| **`tiered`** | Use the tier that will preserve the most version information                       |
```
[Grammar](https://www.diffen.com/difference/That_vs_Which)

**Comment by LilithHafner** [comment]
Bump. Right now, the description of tiered is literally incorrect because the tier which preserves the most information is `all` or 'installed'. This helps.

**Comment by IanButterworth** [comment]
Seems like an improvement. Can you also update the docstrings where this is explained.

**Comment by LilithHafner** [comment]
Yep, I didn't realize this explanation was duplicated. I think I got all two cases now (from `grep -riI "Use the tier"`)

**Review by IanButterworth — APPROVED** [review]
Great, thanks

**Pull request merged**